app/models/cloud_database.rb

```diff
@@ -10,8 +10,39 @@ class CloudDatabase < ApplicationRecord
 
   serialize :extra_attributes
 
+  supports_not :create
   supports_not :delete
 
+  def self.create_cloud_database_queue(userid, ext_management_system, options = {})
+    task_opts = {
+      :action => "creating Cloud Database for user #{userid}",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => self.class.name,
```

Comment:
Since this is a class method you don't need the `self.class` just `name`:
```
ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase.create_cloud_database_queue...
(byebug) name
"ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase"
(byebug) self.class.name
"Class"
```